meteor/lib/api/triggers/actionFilterChainCompilers.ts

```diff
@@ -651,32 +651,54 @@ export function compileAdLibFilter(
 			}
 		}
 
-		let result: IWrappedAdLib[] = []
+		const partMap = new Map<PartId, DBPart>()
+		{
+			if (partFilter?.length) {
+				Parts.find({
+					_id: { $in: partFilter },
+				}).forEach((part) => {
+					partMap.set(part._id, part)
+				})
+			}
+		}
+
+		let adlibs: IWrappedAdLib[] = []
 
 		{
-			result = [
+			adlibs = [
```

Comment:
If you insist on renaming this variable, let's at least rename it to `matchedAdLibs` or `joinedAdLibs`. `adlibs` is very generic, and to me, `result` at least implied "this is where we'll keep the final result of this operation (of matching adlibs)". Now, this implies "this is where we'll keep some adlibs in this operation (of matching adlibs)", obscuring the point of this variable.

Reply:
Good point!
yeah `results` seemed to generic to me, but I get your point too! I'll rename it to `resultingAdlibs`, which should suit us both :)